src/Symfony/Bridge/PhpUnit/Extension/DisableClockMockSubscriber.php

```diff
@@ -25,6 +25,10 @@ public function notify(Finished $event): void
     {
         $test = $event->test();
 
+        if (!method_exists($test, 'metadata')) {
```

Comment:
In which PHPUnit versions does this happen ? This requires at least a comment explaining it (so that we can clean this code when not relevant anymore). And maybe this should be an exception instead as the extension is basically a no-op when this method does not exist, which means it is not actually supported.

Reply:
There are two implementations of the `Test` class: `Phpt` and `TestMethod`. Here we can only handle instances of `TestMethod`. That's why the current versions does use an `instanceof` check instead of calling `method_exists()`. This is now also part of the #58467.